python/python/raphtory/export.py

```diff
@@ -0,0 +1,294 @@
+"""
+Generate a visualisation using matplotlib or pyvis from Raphtory graphs.
+"""
+from pyvis.network import Network
+import networkx as nx
+import pandas as pd
+
+r"""Draw a graph with Pyvis.
+
+.. note::
+
+    Pyvis is a required dependency.
+    If you intend to use this function make sure that
+    you install Pyvis with ``pip install pyvis``
+
+:param graph: A Raphtory graph.
+:param str height: A string defining the height of the graph. By default ``800px`` is set.
+:param str width: A string defining the width of the graph.  By default ``800px`` is set.
+:param str bg_color: A string defining the colour of the graph background. It must be a HTML color code. By default ``#white`` (white) is set.
+:param str font_color: A string defining the colour of the graph font. By default ``"black"`` is set.
+:param str edge_color: A string defining the colour of the edges in the graph. By default ``#000000`` (black) is set.
+:param str shape: An optional string defining what the node looks like. 
+    There are two types of nodes. One type has the label inside of it and the other type has the label underneath it. 
+    The types with the label inside of it are: ellipse, circle, database, box, text. 
+    The ones with the label outside of it are: image, circularImage, diamond, dot, star, triangle, triangleDown, square and icon.
+    By default ``"dot"`` is set.
+:param str node_image: An optional string defining the url of a custom node image. By default an image of a circle is set.
+:param str edge_weight: An optional string defining the name of the property where edge weight is set on your Raphtory graph. By default ``1`` is set.
+:param str edge_label: An optional string defining the name of the property where edge label is set on your Raphtory graph. By default, an empty string as the label is set.
+:param bool notebook: A boolean that is set to True if using jupyter notebook. By default this is set to True.
+
+
+:returns: A pyvis visualisation in static HTML format that is interactive with toggles menu.
+:rtype: IFrame(name, width=self.width, height=self.height)
```

Comment:
this is wrong, it returns a Pyvis graph not an IFrame (it no longer calls show internally)